R/fileDependency.R

```diff
@@ -0,0 +1,26 @@
+#' Create a file dependency that can be accessed by the client using javascript.
+#' @export
+fileDependency <- function(filename, version = '0.0.1'){
+  htmltools::htmlDependency(
+    name = basename(tools:::file_path_sans_ext(filename)),
+    version = version,
+    src = dirname(filename),
+    attachment = basename(filename)
+  )
```

Comment:
I wonder... would there be a way to have htmlwidgets instead of resolving the dependency, just insert a link directly into the page?